src/main/resources/templates/fragments/numberedHeading.html

```diff
@@ -5,8 +5,8 @@
         <title></title>
     </head>
     <body>
-        <h1 id="page-title" th:fragment="numberedHeading" class="heading-xlarge">
-            <span id="page-title-number" class="circle circle-step-large" th:text='${number}'></span>
+        <h1 id="page-title" th:fragment="numberedHeading" class="govuk-heading-l">
+            <!--<span id="page-title-number" class="circle circle-step-large" th:text='${number}'></span>-->
```

Comment:
Should this be commented?

Reply:
ive left this in as the testers are raising a query with rachel whether the circled numbers should be in the headings or not